aries-site/src/pages/foundation/accessibility.mdx

```diff
@@ -65,6 +64,28 @@ While supplemental attributes may need to be implemented and accessibility testi
+3. **Observe through real people.** The best way to understand the problems that users with disabilities face is to see how they navigate in their daily lives. Break down assumptions by learning about disabilities outside of the context of the web, too.
+4. **Conduct testing with real users.** Automated testing tools only catch approximately 30-50% of accessibility violations. Manual testing is essential to thoroughly conducting an accessibility test on a product. Test with a variety of users with disabilities to ensure full coverage. [Bill Tipton](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/bill-tipton-biography.aspx), from the Product Accessibility Office, is an excellent internal resource for usability testing. Bill has used a screen reader in his daily life since 1999 when he lost his eyesight. Please email him or [Bill Bettega](https://hpe.sharepoint.com/sites/F5/CTO/Office/Accessibility/Pages/bill-bettega-biography.aspx) with any of your product testing requests and they will respond when they are able to assist.
+5. **Think of accessibility first.** If accessibility is incorporated into the design process and development lifecycle as early as possible, many violations could be avoided in the future. More research, dialogue, testing and empathy will increase the accessibility of HPE products.
+6. **Be an active advocate.** Testing does not end at the final report of accessibility violations of a product. To further advance accessibility, after testing, you should remain an ally and advocate for the disabled community. Amplifying all voices will lend a hand to making the internet more inclusive.
+
+### Automated Testing Tools
+
+- [AxeDev Tools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US): A browser extension that identifies violations and pinpoints them so you are able to easily see and fix them.
+- [Access Assistant](https://chrome.google.com/webstore/detail/access-assistant/ojiighldhdmahfdnhfdebnpmlbiemdfm/related): An extension tool offered by [Level Access](https://www.levelaccess.com/) that identifies the most common, highest risk accessibility violations on a product. Reports can be saved to monitor compliance over time.
+- [WAVE (Web Accessibility Evaluation Tool) Browser Plugin](https://wave.webaim.org/extension/): Scans webpages to identify missing HTML requirements and color contrast standards.
+- [Lighthouse](https://developers.google.com/web/tools/lighthouse): Integrated into Chrome, this tools is available as a tab after clicking 'Inspect' on a webpage and will generate an accessibility report for the page.
```

Comment:
This is also offered as both a browser tool as well as a can be incorporated into CI/CD. We could point to the DS repo and call out how it is incorporated into our Github actions.